### PR TITLE
[RLlib] Fix connector configurationi for Atari envs.

### PR DESCRIPTION
**Pull request by gjoliver**
Signed-off-by: Jun Gong <jungong@anyscale.com>

## Why are these changes needed?

Make sure connector setup follows existing logics for Atari envs.

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [*] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by ArturNiederfahrenhorst — APPROVED** [review]
@gjoliver Can you fix the linter error? Other than that looks cool! Thanks a lot.

File: rllib/connectors/util.py [hunk 1]
```diff
@@ -27,21 +27,43 @@
...
+        return False
+    return True
+
+
+def __clip_rewards(config: TrainerConfigDict):
+    # Same logic as in RolloutWorker.__init__.
+    # We always clip rewards for Atari games.
+    if config.is_atari:
+        return True
+    return config.clip_rewards
```
Comment (kouroshHakha) [inline rllib/connectors/util.py:50]: nit: `return config.is_atari or config.clip_rewards`

**Review by kouroshHakha — APPROVED** [review]
LGTM, only 1 small nit since you should address linter anyways. Also we need to reflect this new flag somewhere in the documentation. Do we have a atari specific page or sth? Where should it go?

File: rllib/connectors/util.py (hunk 1 above)
Comment (gjoliver) [reply rllib/connectors/util.py]: done

**Comment by gjoliver** [comment]
> LGTM, only 1 small nit since you should address linter anyways. Also we need to reflect this new flag somewhere in the documentation. Do we have a atari specific page or sth? Where should it go?

good question, let's figure that out. put a post out somewhere that Google search can find I think.

**Pull request merged**